package.json

```diff
@@ -15,20 +15,32 @@
     "name": "StrongLoop",
     "url": "https://github.com/strongloop/strong-mesh-models/blob/master/LICENSE.md"
   },
+  "main": "server/server.js",
   "scripts": {
-    "pretest": "jshint . && eslint --ignore-path .jshintignore **/*.js",
-    "test": "tap --tap --stdout --stderr --timeout=300 test/cover.js",
-    "cover": "istanbul cover --print tap --tap --stdout --stderr --timeout=300 test/cover.js"
+    "eslint": "eslint client common server test",
+    "jscs": "jscs client common server test",
+    "test": "tap --tap --stdout --stderr --timeout=300 test/test-*.js",
+    "cover": "test/cover.sh"
+  },
+  "dependencies": {
+    "async": "^0.9.0",
+    "compression": "^1.4.0",
+    "debug": "^2.1.1",
+    "errorhandler": "^1.3.0",
+    "eslint": "^0.15.0",
+    "istanbul": "^0.3.0",
+    "jscs": "^1.11.3",
+    "lodash": "^3.3.1",
+    "loopback": "^2.12.0",
+    "loopback-boot": "^2.6.0",
+    "loopback-datasource-juggler": "^2.18.0",
+    "request": "^2.53.0",
+    "serve-favicon": "^2.2.0",
+    "tap": "^0.6.0"
+  },
+  "optionalDependencies": {
```

Comment:
Doesn't make much sense in this case, but it can help with binary deps that have pure js fallbacks as separate modules.